6.gtex-interpret/scripts/nbconverted/4.visualize-neutrophil-signature-comprarison.r

```diff
@@ -0,0 +1,79 @@
+
+library(ggplot2)
+library(ggrepel)
+suppressPackageStartupMessages(library(dplyr))
+
+file <- file.path('results', 'latent_feature_enrichment_comparison_neutrophil_genesets.tsv')
+geneset_weights_df <- (
+    readr::read_tsv(file,
+                    col_types = readr::cols(.default = readr::col_double(),
+                                            model_type_vae = readr::col_character(),
+                                            variable = readr::col_character(),
+                                            algorithm_vae = readr::col_character(),
+                                            model_type_dae = readr::col_character(),
+                                            algorithm_dae = readr::col_character()))
+    )
+
+head(geneset_weights_df)
+
+color_logic <- ((geneset_weights_df$z_score_vae < -16 | geneset_weights_df$z_score_vae > 18) |
+                (geneset_weights_df$z_score_dae < -17))
+
+ggplot(geneset_weights_df, aes(x = z_score_vae, y = z_score_dae)) +
+  geom_point(size = 0.2) +
+  geom_text_repel(data = subset(geneset_weights_df, color_logic),
+                  arrow = arrow(length = unit(0.01, "npc")),
+                  segment.size = 0.3,
+                  segment.alpha = 0.6,
+                  color = 'red',
+                  size = 2,
+                  fontface = "italic",
+                  point.padding = 0.4,
+                  aes(x = z_score_vae, y = z_score_dae, label = variable)) +
+  xlab("Z Score (VAE)") +
+  ylab("Z Score (DAE)") +
+  theme_bw()
+
```

Comment:
Why are you comparing VAE vs DAE?

Reply:
These are two features that appear to have good scores capturing neutrophil signatures